TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/fragment/PostFragment.java

```diff
@@ -0,0 +1,375 @@
+        setGalleryButton(view);
+        setDraftButton(view);
+        setHashtagButton(view);
+
+        return view;
+    }
+
+    private void setTweet(View view) {
+        // Get status
+        TwitterStatus status = null;
+        status = mReplyStatus != null ? mReplyStatus : status;
+        status = mQuoteStatus != null ? mQuoteStatus : status;
+
+        // Check if status exist
+        if (status == null) {
+            view.findViewById(R.id.scroll_tweet).setVisibility(View.GONE);
+            return;
+        }
+
+        // Render tweet
+        TweetRow tweetRow = new TweetRow(view);
+        tweetRow.initVisibilities();
+        tweetRow.setStatus(status);
+        tweetRow.setUserName();
+        tweetRow.setScreenName();
+        tweetRow.setUserIcon();
+        tweetRow.setTweetText();
+        tweetRow.setRelativeTime();
+        tweetRow.setAbsoluteTime();
+        tweetRow.setVia();
+        tweetRow.setRtFavCount();
+        tweetRow.setRetweetedBy();
+        tweetRow.setMarks();
+        tweetRow.setThumbnail();
+        tweetRow.setQuoteTweet();
+        tweetRow.setBackgroundColor();
+    }
+
+    private void setPostEdit(View view) {
+        // Find
+        mTextCount = view.findViewById(R.id.text_count);
+        mPostEdit = view.findViewById(R.id.edit_post);
+
+        // When edit text changed
+        mPostEdit.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+                int length = s.toString().length();
+                int color = length > 140 ? Color.RED : COLOR_TEXT;
+
+                // Change count
+                mTextCount.setText(Integer.toString(140 - length));
+                mTextCount.setTextColor(color);
+            }
+
+            @Override
+            public void afterTextChanged(Editable s) {
+            }
+
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+            }
+        });
+    }
+
+    private void setIntentData() {
+        // Check external
+        if (mExternalText != null) {
+            mPostEdit.setText(mExternalText);
+            mPostEdit.setSelection(mExternalText.length());
+        }
+
+        // Check prefix
+        if (mTweetPrefix != null) {
+            mTweetPrefix += " ";
+            mPostEdit.setText(mTweetPrefix);
+            mPostEdit.setSelection(mTweetPrefix.length());
+        }
+
+        // Check suffix
+        if (mTweetSuffix != null) {
+            mPostEdit.setText("\n" + mTweetSuffix);
+        }
+
+        // Check reply
+        if (mReplyStatus != null) {
+            // Append reply user
+            String replyScreen = mReplyStatus.getUser().getScreenName();
+            String str = "@" + replyScreen + " ";
+
+            // Append mention user
+            for (TwitterUserMentionEntity mention : mReplyStatus.getUserMentionEntities()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS135)